From 2c14a56ca5cc270866348b25732256f164a203c4 Mon Sep 17 00:00:00 2001
From: Lim Siew Hoon <siew.hoon.lim@intel.com>
Date: Mon, 30 Dec 2024 08:25:53 +0800
Subject: [PATCH 7/7] [VP] Fix GST VA Video output of Color conversion - UYVY
 -> RGB32 is corrupted

Fix GST VA Video output of Color conversion - UYVY -> RGB32 is corrupted.

JIRA: VSMGWL-77286

Signed-off-by: Chng, Seng Guan <seng.guan.chng@intel.com>
Signed-off-by: Lim Siew Hoon <siew.hoon.lim@intel.com>
---
 media_driver/linux/common/ddi/media_libva.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/media_driver/linux/common/ddi/media_libva.cpp b/media_driver/linux/common/ddi/media_libva.cpp
index 777cb4d8e..d3fd88cff 100755
--- a/media_driver/linux/common/ddi/media_libva.cpp
+++ b/media_driver/linux/common/ddi/media_libva.cpp
@@ -4609,7 +4609,6 @@ VAStatus DdiMedia_CreateImage(
     gmmParams.Type              = RESOURCE_2D;
     gmmParams.Flags.Gpu.Video   = true;
     gmmParams.Format            = mediaCtx->m_caps->ConvertFourccToGmmFmt(format->fourcc);
-    gmmParams.Flags.Info.Linear = 1;
 
     if (gmmParams.Format == GMM_FORMAT_INVALID)
     {
-- 
2.43.0

